src/download.c

```diff
@@ -301,6 +301,7 @@ int untar_full_download(void *data)
 	if (err) {
 		printf("ignoring tar extract failure for fullfile %s.tar (ret %d)\n",
 		       file->hash, err);
+		err = 0;
```

Comment:
Could go with "Problem extracting tar file, attempting redownload" and make sure we are cleaning the tar file (it looked like download.c didn't do this for full files just packs).